Dockerfile

```diff
@@ -1,53 +1,97 @@
-ARG BASE_IMAGE=python
-ARG PYTHON_VERSION=3.9.4
-ARG IMAGE_VARIANT=slim
+# Dockerfile for Parking Permits backend
 
-# ==============================
-FROM ${BASE_IMAGE}:${PYTHON_VERSION}-${IMAGE_VARIANT} AS base_stage
-# ==============================
+FROM registry.access.redhat.com/ubi8/python-38 as appbase
 
-RUN groupadd --system --gid 2000 appgroup && \
-    useradd  --system --gid      appgroup --create-home --uid 3000 appuser
+USER root
+
+RUN rm /etc/rhsm-host
+
+ARG LOCAL_REDHAT_USERNAME
+ARG LOCAL_REDHAT_PASSWORD
+ARG BUILD_MODE
+
+RUN if [ "x$BUILD_MODE" = "xlocal" ] ;\
+    then \
+        subscription-manager register --username $LOCAL_REDHAT_USERNAME --password $LOCAL_REDHAT_PASSWORD --auto-attach; \
+    else \
+        subscription-manager register --username ${REDHAT_USERNAME} --password ${REDHAT_PASSWORD} --auto-attach; \
+    fi
+
+RUN subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
+RUN yum -y update
+
+RUN rpm -Uvh https://download.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+
+RUN yum install -y gdal
+
+RUN useradd -ms /bin/bash -d /app parking_permits
+# Statics are kept inside container image for serving using whitenoise
+RUN mkdir -p /srv/static && chown parking_permits /srv/static && chown parking_permits /opt/app-root/bin
+
+RUN chown parking_permits /opt/app-root/lib/python3.8/site-packages
+RUN chown parking_permits /opt/app-root/lib/python3.8/site-packages/*
+RUN pip install --upgrade pip
+
+# Can be used to inquire about running app
+# eg. by running `echo $APP_NAME`
+ENV APP_NAME parking_permits
 
 WORKDIR /app
 
-EXPOSE 8000
+COPY . /app/
+
+RUN subscription-manager remove --all
+
+# Served by whitenoise middleware
+ENV STATIC_ROOT /srv/static
 
 # Add tini init system https://github.com/krallin/tini
 ENV TINI_VERSION v0.19.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
 RUN chmod +x /tini
 
-ENV PYTHONDONTWRITEBYTECODE true
-ENV PYTHONUNBUFFERED true
+ENV PYTHONDONTWRITEBYTECODE True
+ENV PYTHONUNBUFFERED True
+ENV PYTHONUSERBASE /pythonbase
+
+# Copy and install requirements files to image
+COPY requirements.txt ./
+
+RUN pip install --no-cache-dir uwsgi
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+ENV DEBUG=True
+
+# RUN python manage.py collectstatic --noinput
```

Comment:
On line 46 you are serving the static files from `/srv/static`. So if this line is removed I guess it's not collecting the static files automatically and placing them to `/srv/static`